tests/nested/manual/core20-new-snapd-does-not-break-old-initrd/task.yaml

```diff
@@ -0,0 +1,86 @@
+summary: verify that new snapd's do not break old snap-bootstrap/kernel initrds
+
+systems: [ubuntu-20.04-64]
+
+# we have two variants here:
+#
+# 1. we start with stable kernel + stable snapd -> refresh to new snapd
+# 2. we start with stable kernel + new snapd
+#
+# and then in both cases we then trigger a reseal operation and reboot to make
+# sure that the old snap-bootstrap/initrd in the stable kernel can still unlock
+# the encrypted partitions
+
+environment:
+  NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/ubuntu-core-20-amd64.model
+
+  # don't bundle the snapd snap via nested.sh's machinery, instead we will
+  # side-load our snapd snap built from this branch into the image via
+  # extra-snaps since we still want to use MS keys and such for this image, and
+  # defining this to be true brings with it i.e. snakeoil keys in the OVMF
+  # firmware for example
+  NESTED_BUILD_SNAPD_FROM_CURRENT: false
+
+  # we want snaps from the stable channel by default with the exception of the
+  # snapd snap which we will conditionally repack as per NESTED_BUILD_SNAPD_FROM_CURRENT
+  NESTED_CORE_CHANNEL: stable
+
+  START_SNAPD_VERSION/startwithnew: new
+  START_SNAPD_VERSION/startwithstable: old
+
+  NESTED_IMAGE_ID: uc20-breakages-testing-$START_SNAPD_VERSION
+
+  # all variants need encryption turned on
+  NESTED_ENABLE_TPM: true
+  NESTED_ENABLE_SECURE_BOOT: true
+
+  INITIAL_KERNEL_REV_URL: https://storage.googleapis.com/snapd-spread-tests/snaps/pc-kernel_838.snap
```

Comment:
at some point once we have decided a way forward also snapd might need to come from a precise version